uxarray/io/_geos.py

```diff
@@ -0,0 +1,59 @@
+import xarray as xr
+import numpy as np
+
+from uxarray.constants import INT_DTYPE
+from uxarray.conventions import ugrid
+
+
+def _read_geos_cs(in_ds: xr.Dataset):
```

Comment:
Does this grid format support cartesian?

Reply:
I don't believe so. 

@mathomp4 Are the coordinates typically spherical or are they every provided as Cartesian?

Reply:
Umm. I'm going to mention @bena-nasa here. He's our grids guru. I mainly know: our native CS output looks like this when I ask for it 😄 

We can also put out lat-Lon output that is of a form everything can read. But the CS output is the weird one. (And I think our restarts might be a slightly different CS unstructured grid style...)

Reply:
Resolving this comment for now, as we don't need Cartesian coordinates anyways. We can always add support for parsing them if it turns out they are supported by GEOS in the future.